### PR TITLE
fix: download speed & integrate transcripts

### LINKED ISSUE #217: low download speed
I often see low download speeds, around 512kbs. Do you know what could be the problem? These are random cases, but lately it happens quite often. Use programmatically

### PR DESCRIPTION
**Pull request by Davincible**
# Description

Fix the download speed. Get ~300+ Mb/s~ full gigabit speed now.

- Switched to Android client
- Added multi-processing downloads

Code is very rough, could use some cleanup. But works for my purposes. Feel free to push to this branch directly to clean up.

All tests pass.

## Issues to fix

#217 #282

### CODE REVIEW

**Comment by Davincible** [comment]
@corny

File: utils.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+func getFirstKeyJSON(j *sjson.Json) *sjson.Json {
+	m, err := j.Map()
+	if err != nil {
+		return j
+	}
+
+	for key := range m {
+		fmt.Println("returning item")
```
Comment (corny) [inline utils.go:40]: please remove `fmt.Println` statements

**Review by corny — CHANGES_REQUESTED** [review]

**Comment by denis-kilchichakov** [comment]
Can't wait this PR to be merged - low speed makes this library barely usable :/

**Comment by pytomtoto** [comment]
nice job

**Comment by Forsigg** [comment]
> Can't wait this PR to be merged - low speed makes this library barely usable :/

Yep, fully agree. I'm already use this fork in my pet project, works great. Download speed has increased greatly.
Thanks @Davincible for this fix!

**Comment by Davincible** [comment]
@corny

**Comment by corny** [comment]
Thanks a lot for you work! I've just released a new version.

**Pull request merged**